### PR TITLE
fix-function-declaration-and-Error-log

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: utils/dotcms/utilities.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ class CustomError extends Error {
 
 module.exports = {
     isPage,
+    isAPIRequest,
```
Comment (freddyucv) [inline utils/dotcms/utilities.js]: why this change?

Comment (alfredo-dotcms) [reply utils/dotcms/utilities.js]: the app breaks if not exported (it is being used on other file)
